libs/types/src/tabulation.ts

```diff
@@ -76,7 +76,7 @@ export interface Filter {
  * Attributes that always exist for every cast vote record.
  */
 export interface CastVoteRecordAttributes {
-  readonly ballotStyleGroupId: BallotStyleGroupId;
+  readonly ballotStyleGroupId: Id;
```

Comment:
Why this change?

Reply:
I forget exactly but I think there were insurmountable other issues if this isn't a string as all other fields here are strings and that is utilized in other parts of the code base that generalize the keys